internal/pkg/template/workload.go

```diff
@@ -488,6 +488,7 @@ type WorkloadOpts struct {
 	// Additional options for request driven web service templates.
 	StartCommand      *string
 	EnableHealthCheck bool
+	Tracing           string
```

Comment:
nit: What do you think of moving this into something like this:
```suggestion
	Observability           ObservabilityOpts
```
```go
type ObservabilityOpts struct {
    Tracing string
}
```

Reply:
Updated!